packages/customer-effort-score/src/index.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * External dependencies
+ */
+import { useState } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+
+/**
+ * Use `CustomerEffortScore` to gather a customer effort score.
+ *
+ * NOTE: This should live in @woocommerce/customer-effort-score to allow
+ * reuse.
+ *
+ * @param {Object}   props               Component props.
+ * @param {Function} props.trackCallback Function to call when the modal is activated.
+ * @param {boolean}  props.visible       Whether or not the tracks modal is visible.
+ * @param {Function} props.toggleVisible Callback to toggle the visible prop.
+ * @param {string}   props.label         The label displayed in the modal.
+ */
+function CustomerEffortScore( {
+	trackCallback,
+	visible,
+	toggleVisible,
+	label,
+} ) {
+	const [ score, setScore ] = useState( 0 );
+
+	if ( ! visible ) {
+		return null;
+	}
+
+	function close() {
+		setScore( 3 ); // TODO let this happen in the UI
+
+		toggleVisible();
+		trackCallback( score );
+	}
+
+	return (
+		<p className="customer-effort-score_modal">
+			{ label } <button onClick={ close }>Click me</button>
```

Comment:
So is the idea the final design here will have multiple buttons that all call `close()`? Will the design be tightly coupled to what @elizaan36 has created or are we going to make it a more config-based render?

Reply:
I think it should be @elizaan36 's design, not abstracting things just for the sake of it otherwise we're trading configurability for the ability to just use the CES package.